Tests/SQLiteKitTests/SQLiteKitTests.swift

```diff
@@ -5,102 +5,112 @@ import XCTest
 import SQLiteNIO
 import SQLKit
 
-class SQLiteKitTests: XCTestCase {
+final class SQLiteKitTests: XCTestCase {
```

Comment:
Why final in _tests_?

Reply:
All classes should always be `final` unless they're explicitly intended for subclassing (not forgetting that subclassing has been soft-deprecated in Swift for a long time and subclassing `XCTestCase` is the only legitimate use of it I've yet run into in the server ecosystem). A `final` class can be better optimized, can be used covariantly and contravariantly when referring to `Self` (I won't pretend that I remember the exact rules involved in that one; if something about that seems incorrect, it probably is), and IMO is just cleaner.